documentation/src/main/resources/_posts/2023-08-23-ditto-benchmark.md

```diff
@@ -0,0 +1,465 @@
+---
+title: "Eclipse Ditto Benchmark"
+published: false
+permalink: 2023-08-23-ditto-benchmark.html
+layout: post
+author: vvasilevbosch
```

Comment:
@vvasilevbosch if you add a new `author`, you must also update the file `_data/authors.yml` with that username - otherwise the blogentry will not render the author correctly.